setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = django-push-notifications
-version = 1.5.0
+version = 1.6.0
```

Comment:
Please do not bump the version, this is done during releases.

Reply:
Done [5084ff16dd9c855cd087048b84eec9692592c228]